mappings/net/minecraft/client/render/block/entity/model/DragonHeadEntityModel.mapping

```diff
@@ -1,4 +1,4 @@
-CLASS net/minecraft/unmapped/C_ksfxmjly net/minecraft/client/render/entity/model/DragonHeadEntityModel
+CLASS net/minecraft/unmapped/C_ksfxmjly net/minecraft/client/render/block/entity/model/DragonHeadEntityModel
```

Comment:
This should be in `net/minecraft/client/render/entity/model/` like `net/minecraft/client/render/entity/model/SkullEntityModel`

Reply:
It's a model that is only used on BlockEntities though, so having them on the entity directory doesn't really make sense

Reply:
I found out that my restructuration was flawed, so I did it again but better